src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java

```diff
@@ -120,8 +120,7 @@ public AboutToStartOrSubmitCallbackResponse populateJudgeOrAdminDraftOrderCustom
         );
         Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
         DraftOrder selectedOrder = draftAnOrderService.getSelectedDraftOrderDetails(caseData);
-        if (selectedOrder != null && (CreateSelectOrderOptionsEnum.blankOrderOrDirections.equals(selectedOrder.getOrderType())
-            || CreateSelectOrderOptionsEnum.blankOrderOrDirectionsWithdraw.equals(selectedOrder.getOrderType()))
+        if (selectedOrder != null && (CreateSelectOrderOptionsEnum.blankOrderOrDirections.equals(selectedOrder.getOrderType()))
```

Comment:
You dont need these many bracket.
How we are managing the existing features of withdraw order?

Reply:
just brackets now. got the ticket https://tools.hmcts.net/jira/browse/PRL-3091

Reply:
Withdraw order features are available as part of new c21 order screen